tests/integration/test_system_containers_images.sh

```diff
@@ -20,7 +20,7 @@ setup () {
 
 teardown () {
     set +o pipefail
-
+    ${ATOMIC} -y containers delete busybox
```

Comment:
this will actually cause a temporary problem, if the tests succedd running, trying to fix it at the moment